v4 (redux)

h-vetinari opened the pull request:
Clean up of #12. The commits around the windows testing could still be squashed, but for now I just did the minimal surgery to get an acceptable git history.

Closes #8 
Closes #9 
Closes #10 
Closes #11 
Closes #12 
Closes #13

0xbe7a committed:
v4

Co-Authored-By: H. Vetinari <h.vetinari@gmx.com>

0xbe7a committed:
libdir

0xbe7a committed:
set target

0xbe7a committed:
win tests

h-vetinari committed:
translate to v1; add osx-arm64

Co-Authored-By: Bela Stoyan <git@be7a.de>

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

h-vetinari commented on recipe/recipe.yaml:
```diff
@@ -0,0 +1,61 @@
+      files:
+        exists:
+          - "**/lib/pkgconfig/imagequant.pc"
+        not_exists:
+          # absence of static libraries
+          - lib/libimagequant.a
```
In the meantime, https://github.com/prefix-dev/rattler-build/issues/915 was fixed, so we can do better in the translation of the original recipe.

h-vetinari committed:
switch osx-arm to native builds; add linux-aarch64

h-vetinari committed:
MNT: Re-rendered with conda-smithy 2026.5.29 and conda-forge-pinning 2026.06.02.12.03.26

Other tools:
- conda-build 26.5.0
- rattler-build 0.65.1
- rattler-build-conda-compat 1.4.14

h-vetinari committed:
update expected license file name
